include/wx/gtk/setup.h

```diff
@@ -1057,7 +1057,7 @@
 //
 // Default is 1.
 //
-// Recommended setting: 1 but can be safely set to 0 except for wxUniv where it
+// Recommended setting: 1 but can be safely set to 0 except for wxUniv where
 //                      it used by wxComboBox
```

Comment:
I think this "it" should be kept and the next one replaced with "is", otherwise the sentence is missing the verb.

Also, `setup.h` files are (partially) auto-generated, so please update `include/wx/setup_inc.h` and rerun `./build/update-setup-h` if you need to make any changes to them in the future. TIA!